README.md

```diff
@@ -119,6 +119,95 @@ $ cat /usr/share/qm/containers.conf | grep oom_score_adj
 oom_score_adj = 750
 ```
 
+### Adjusting OOM Score for Nested Containers
+
+You can customize the OOM score adjustment for nested containers running inside QM. This requires both nested container configurations and QM capability adjustments.
+
+#### Step 1: Create Nested Container Configurations
+
+Create nested container configurations in `/etc/qm/containers/systemd/`:
+
+**Create:** `/etc/qm/containers/systemd/nested.container`
+
+```ini
+[Container]
+ContainerName=nested
+Image=alpine:latest
+Exec=sleep 1d
+Network=none
+
+[Service]
+OOMScoreAdjust=1000
+```
+
+**Create:** `/etc/qm/containers/systemd/nested2.container`
+
+```ini
+[Container]
+ContainerName=nested2
+Image=alpine:latest
+Exec=sleep 1d
+Network=none
+
+[Service]
+OOMScoreAdjust=100
+```
+
+#### Step 2: Add Required Capabilities to QM
```

Comment:
is it only for nested containers? 
Consider explaining better how to override options defined by qm.container
I assume it will be under Line nu. 304 
Configuring QM